### ISSUE TITLE
Should forfeits count as a gov or an opp?

### ISSUE THREAD

**Issue by roddajohn** [MIT-Tab/mit-tab#269]
General question — not even sure what SOP is, but thought I'd raise the question

**Comment by BenMusch** [comment]
For in-rounds I think they definitely shouldn't. I'm pretty sure they shouldn't for outs either, but just need to think through whether there could be some bizarre choice policies where this should matter

**Comment by roddajohn** [comment]
Yes I agree, esp because BYEs don't (right?).  Ran into this at Penn where end result was a team OPPed 3 times and govved once which is technically legal but if it had counted the R1 forfiet as without gov/opp than I think it would have been 2/2 depending on the rest of the bracket.

**Comment by roddajohn** [comment]
And for outs, considering the code currently doesn't figure out who has choice, just gives the users an option to indicate who does, it wouldn't affect outs (tournament can decide how they want to handle and then toggle accordingly)

**Comment by BenMusch** [comment]
Yup. I think there's a `num_govs` function in `tab_logic` which we can just modify to make this change. Worth noting that there's a chance tests fail after this as they compare the output of the pairing algorithm vs. the "correct" pairings which have already been generated

**Comment by roddajohn** [comment]
Oh that's a good catch re the tests, I'll be careful when making this change then.
